frappe/integrations/doctype/social_login_keys/social_login_keys.py

```diff
@@ -17,10 +24,14 @@ def validate_frappe_server_url(self):
 		if self.frappe_server_url:
 			if self.frappe_server_url.endswith('/'):
 				self.frappe_server_url = self.frappe_server_url[:-1]
-			import requests
+
 			try:
-				r = requests.get(self.frappe_server_url + "/api/method/frappe.handler.version", timeout=5)
+				frappe_server_hostname = urlparse(self.frappe_server_url).netloc
 			except:
-				frappe.throw(_("Unable to make request to the Frappe Server URL"))
-			if r.status_code != 200:
 				frappe.throw(_("Check Frappe Server URL"))
+
+			if socket.gethostname() != frappe_server_hostname:
```

Comment:
@revant 

I think it should be verified if frappe_server_hostname is in local.conf.domais in case of a configuration of multidomains (https://frappe.github.io/frappe/user/en/bench/guides/adding-custom-domains)